wp-job-manager-template.php

```diff
@@ -1371,5 +1400,41 @@ function get_the_job_salary_unit( $post = null ) {
 	 * @param string  $job_unit
 	 * @param WP_Post $post
 	 */
-	return apply_filters( 'wpjm_job_salary_unit', 'YEAR', $post );
+	return apply_filters( 'wpjm_job_salary_unit', $job_unit, $post );
+}
+
+/**
+ * Get the display text for the job salary unit
+ *
+ * @since 1.37.0
+ * @param int|WP_Post|null $post (default: null).
+ * @return string|null
+ */
+function get_the_job_salary_unit_display_text( $post = null ) {
+	$job_unit = get_the_job_salary_unit( $post );
+
+	$translated_options = [
```

Comment:
I'm realizing that this is the 4th place where this list has been duplicated. I wonder if we should extract them out? As a bonus, we could even make it filterable 🤷‍♂️

Reply:
Yep, I'd really like to avoid duplicating that list, but, at the same time, I don't have an idea of where to put it.

Do you have any suggestions? Maybe a new class or something?

Reply:
I'm not sure it warrants a whole new class. What about just throwing it in `wp-job-manager-functions.php`? 🤷‍♂️

Reply:
Good idea, done in 81e8252cc7ef622b9e4de2d38dfb4de1a316536c.